test/test-pages/webmd-1/expected.html

```diff
@@ -1,5 +1,6 @@
 <div id="readability-page-1" class="page">
     <div id="textArea">
+        <p>Reviewed by <a onclick="return sl(this,'','prog-lnk');" href="http://www.webmd.com/hansa-bhargava">Hansa D. Bhargava, MD</a> </p>
```

Comment:
These types of small divs etc. are now getting "promoted" to `P` tags, which means they no longer get cleaned conditionally. This has unfortunate effects e.g. in the wikipedia case, where all the "see also" text etc. now ends up as part of the main body text, and even gets used for the excerpt.

Is this just because we're dropping the classes immediately or something, or because the change from `div` to `p` means we no longer critically check the node? It would be nice not to regress this, because otherwise this looks like a positive change. :-)